src/commands/uib/exams.ts

```diff
@@ -27,44 +26,51 @@ class ExamCommand extends Command {
 
   run = async (message: CommandoMessage, { subject }: { subject: string }): Promise<Message | Message[]> => {
     const inputSubject = subject.toUpperCase().trim();
+    const courseId = inputSubject.toLowerCase();
 
     if (inputSubject === "") {
       return await message.reply("You need to specify the subject to ask about");
     }
 
-    const course = await getExams(inputSubject, message);
+    const course = await getCourse(inputSubject, message);
     if (course === undefined) {
       return await message.reply(`Sorry, no course with the code ${inputSubject} found... try again`);
     }
 
+    if (course.exams?.length === 0 ?? true) {
+      return await message.reply(`It doesn't seem to be any exams for ${inputSubject}.`);
+    }
+
     const embed = new MessageEmbed()
       .setColor("#0099ff")
-      .setTitle(course?.id)
-      .setURL(course?.url ?? "")
-      .setDescription(course?.name);
+      .setTitle(courseId)
+      .setURL(course.url ?? "")
+      .setDescription(course.name_en);
 
     this.buildExamEmbed(course, embed);
 
     return await message.say(embed);
   };
 
   buildExamEmbed = (course: Course, embed: MessageEmbed): void => {
-    for (const exam of course.exams) {
-      let fields = [];
-      if (Object.prototype.hasOwnProperty.call(exam, "date")) {
-        fields.push({ name: "Dato", value: exam.date, inline: true });
-      }
-      if (Object.prototype.hasOwnProperty.call(exam, "duration")) {
-        fields.push({ name: "Varighet", value: exam.duration, inline: true });
-      }
-      // if (Object.prototype.hasOwnProperty.call(exam, "location")) {
-      //   fields.push({ name: "Location", value: exam.location, inline: true });
-      // }
-      if (Object.prototype.hasOwnProperty.call(exam, "system")) {
-        fields.push({ name: "Eksamensystem", value: exam.system, inline: true });
+    if (course.exams) {
```

Comment:
Har denne plutselig begynt å kræsje her eller?

Reply:
Jeg endret variablen til en nullable, ettersom den ikke alltid har en verdi og det virket mer ryddig enn en tom liste. Gjorde koden litt mer tydelig for å reflektere det.